R/combined.R

```diff
@@ -286,17 +293,40 @@ combined_aggregate_data <- function(data) {
 }
 
 
-combined_aggregate_rt <- function(rt, samples) {
+aggregate_severity <- function(severity, samples) {
```

Comment:
`combined_aggregate_severity` would be more consistent

Reply:
changed